pkg/storaged/dialogx.jsx

```diff
@@ -897,9 +897,13 @@ export const SizeSlider = (tag, title, options) => {
     };
 };
 
+function add_para(parts, text) {
+    parts.push(<p key={text}>{text}</p>);
```

Comment:
That looks a bit silly, but the [documentation](https://reactjs.org/docs/reconciliation.html#keys) apparently makes no restrictions about key length or content.